app/models/post.rb

```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true
+  validates :title, length: { maximum: 250 }
```

Comment:
[OPTIONAL] There is a repetition with `validates :title`. Following the DRY rule, I recommend adding length and presence on the same line.
```rb
validates :title, presence: true, length: { maximum: 250 }
```

Reply:
Thanks for your feedback